docs/remote/containers.md

```diff
@@ -231,6 +235,36 @@ Selecting the **Remote-Containers: Add Development Container Configuration Files
 
 To learn more about creating `devcontainer.json` files, see [Create a Development Container](/docs/remote/create-dev-container.md).
 
+## Dev container features
+
+Dev container features provide a smooth path for customizing your container definitions.
+
+When you use **Remote-Containers: Add Development Container Configuration Files**, you're presented a list of scripts to customize the existing dev container configurations, such as installing Git or the Azure CLI:
+
+![Dev container features in Command Palette](images/containers/container-features.png)
+
+When you rebuild and reopen in your container, the features you selected will be available in your `devcontainer.json`:
+
+```json
+"features": {
+		"github-cli": "latest"
+	}
+```
+
+The features are sourced from the [script library](https://github.com/microsoft/vscode-dev-containers/tree/main/script-library) in the vscode-dev-containers repo.
+
+## Pre-building
```

Comment:
Since "Pre-building" is unfortunatley getting used in a slightly different way in Codespaces, I'd suggest we be specific and say "Pre-building dev container images"